__tests__/app.test.js

```diff
@@ -191,4 +191,63 @@ describe("/api/articles/:article_id/comments", () => {
         expect(body.comment).toHaveProperty("body", "test comment");
       });
   });
-})
+});
+
+describe("/api/articles/:article_id", () => {
+  test("PATCH:200 responds with the updated article and updates the article votes by 1", () => {
+    return request(app)
+      .patch("/api/articles/1")
+      .send({ inc_votes: 1 })
+      .expect(200)
+      .then(({ body }) => {
+        const {article} =body
+        expect(article).toHaveProperty("article_id", 1);
+        expect(article).toHaveProperty("title", expect.any(String));
+        expect(article).toHaveProperty("author", expect.any(String));
+        expect(article).toHaveProperty("body", expect.any(String));
+        expect(article).toHaveProperty("topic", expect.any(String));
+        expect(article).toHaveProperty("created_at", expect.any(String));
+        expect(article).toHaveProperty("votes", 101);
+        expect(article).toHaveProperty("article_img_url", expect.any(String));
+      });
+  });
+
+  test("PATCH:200 responds with the updated article and updates the article votes by -100", () => {
+    return request(app)
+      .patch("/api/articles/1")
+      .send({ inc_votes: -100 })
+      .expect(200)
+      .then(({ body }) => {
+        const {article} =body
+        expect(article).toHaveProperty("article_id", 1);
+        expect(article).toHaveProperty("title", expect.any(String));
+        expect(article).toHaveProperty("author", expect.any(String));
+        expect(article).toHaveProperty("body", expect.any(String));
+        expect(article).toHaveProperty("topic", expect.any(String));
+        expect(article).toHaveProperty("created_at", expect.any(String));
+        expect(article).toHaveProperty("votes", 0);
+        expect(article).toHaveProperty("article_img_url", expect.any(String));
+      });
```

Comment:
When testing properties inside objects there are matchers that will make your code much nicer. 

Check out the jest docs for a list of available matchers.